lms/templates/dashboard.html

```diff
@@ -181,6 +181,17 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
       </ul>
     </section>
   </section>
+  % if program_credentials:
+    <div class="wrapper-xseries-certificates">
+      <p class="title">${_("XSeries Program Certificates")}</p>
+      <p class="copy">${_("You have received a certificate for the following XSeries programs:")}</p>
+      <ul>
+      % for program_detail in program_credentials:
```

Comment:
Use `for program_credential in program_credentials`. It reads better and makes more sense than `program_detail`.